docs/User Interface/Applications/index.md

```diff
@@ -52,7 +52,7 @@ Service Provider options appear when an application has requested to be an App S
 ### Developer Contact Info
 | Property | Definition |
 |----------|---------|
-| Vendor | The name of the developer to contact with regards to this application. |
+| Vendor | The name of the developer to contact in regards to this application. |
```

Comment:
```suggestion
| Vendor | The name of the developer to contact regarding this application. |
```
There's debate whether "in/with" or "regards/regard" are correct, so "regarding" seems like the safest choice here.